lib/mrtrix3/app.py

```diff
@@ -192,6 +192,15 @@ def _execute(module): #pylint: disable=unused-variable
   CMDLINE.print_citation_warning()
 
   return_code = 0
+
+  cli_parse_only = os.getenv("MRTRIX_CLI_PARSE_ONLY")
+  if cli_parse_only == "1":
+    warn(
+      "Quitting after parsing command-line arguments successfully because "
+      "environment variable 'MRTRIX_CLI_PARSE_ONLY' is set to '1'"
+    )
```

Comment:
```suggestion
  cli_parse_only = os.getenv('MRTRIX_CLI_PARSE_ONLY')
  if cli_parse_only:
    try:
      if cli_parse_only.lower() in ['yes', 'true'] or int(cli_parse_only):
        info(
          'Quitting after parsing command-line arguments successfully due to '
          'environment variable "MRTRIX_CLI_PARSE_ONLY"'
        )
        sys.exit(return_code)
    except ValueError:
      warn('Potentially corrupt environment variable "MRTRIX_CLI_PARSE_ONLY" '
           '= " + cli_parse_only + '; ignoring')
```

Needs testing.

-   The environment variable should be interpreted as a boolean value (ideally the *MRtrix3* interpretation of strings as bools should be functionalised somewhere for Python as it is for C++).

-   Should be INFO level rather than WARN.